otel.yml

```
@@ -21,6 +21,9 @@ extensions:
     check_interval: 1s
     limit_mib: 700
     spike_limit_mib: 180
+  health_check:
+    endpoint: "localhost:13133"
+    path: "/health/status"
 
 service:
   extensions: [memory_limiter]
```

Should we add it in the pipeline too?

```suggestion
  extensions: [health_check, memory_limiter]
```

---

Yes, duh 🤦 - thanks for catching!